Fun.Build/StageContextExtensions.fs

```diff
@@ -499,6 +499,23 @@ module StageContextExtensions =
             | ValueSome(StageParent.Stage s) -> s.GetAllCmdArgs()
```

Comment:
Can we add some comments on the **GetAllCmdArgs** to identify it will not return the remaing ones?